### PR TITLE
Fix/2524/solve-map-loading-of-compressed-files

### PR DESCRIPTION
**Pull request by Hall-Ma**
# Compressed files not being loaded

Issue: #2524

## Description

Compressed files are not accepted after the 1.83.1 patch because of a missing string cast.

### CODE REVIEW

**Review by ce-bo — APPROVED** [review]

**Review by BridgeAR — COMMENTED** [review]
It would be great to have a test to guard against similar issues in the future. :-)

**Comment by ce-bo** [comment]
Please adjust CHANGELOG.md

File: visualization/package.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
 		"test:auto": "npm test -- --watch",
 		"test:autoNoCoverage": "npm run test:auto -- --coverage false",
 		"test:updateSnaps": "npm test -- --updateSnapshot",
-		"e2e": "cross-env TEST_TYPE=\"e2e\" jest --config ./jest.config.json .*\\.e2e\\.ts --coverage false",
+		"e2e": "cross-env TEST_TYPE=\"e2e\" jest --config ./jest.config.json .fileChooser.e2e.ts --coverage false",
```
Comment (ce-bo) [inline visualization/package.json:74]: Please revert this change.

**Review by ce-bo — APPROVED** [review]
LGTM. In the next step, after hotfix release, we should manage to generate an up-to-date .gz file during our build pipeline. This file can then be used in an e2e test for testing the upload of zipped files.
